hack/generated/controllers/crd_cosmosdb_test.go

```diff
@@ -6,7 +6,6 @@ Licensed under the MIT license.
 package controllers_test
 
 import (
-	"context"
 	"testing"
 
 	. "github.com/onsi/gomega"
```

Comment:
Is the expectation that these tests don't need to use any of the built-in gomega assertions because we've got wrappers on `testContext`? And if they _do_ need an assertion then it would be via `testContext.G`?

If so, maybe remove this import from all the tests? Hopefully the fact this isn't imported at all will help guide people to using the right assertions

Reply:
It’s only `Expect`/`Eventually` that come off `testContext`, you still need `gomega` imported for all the actual assertions.

Reply:
Ahh, I see.